ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -258,12 +258,14 @@ private MainnetProtocolSpecs() {}
 
   private static TransactionReceipt frontierTransactionReceiptFactory(
       final TransactionProcessor.Result result, final WorldState worldState, final long gasUsed) {
-    return new TransactionReceipt(worldState.rootHash(), gasUsed, result.getLogs());
+    return new TransactionReceipt(
+        worldState.rootHash(), gasUsed, result.getLogs(), result.getRevertReason());
```

Comment:
This is probably good belt and braces stuff, but for the record - there are no revert reasons in frontier, the REVERT opcode was only added in Byzantium.

Reply:
Fixed